vizro-core/docs/pages/user-guides/navigation.md

```diff
@@ -250,6 +402,52 @@ You can alter the icons used by specifying the name of the icon in the [Google M
             ),
         )
         ```
+    === "app.py"
+
+        ```{.python pycafe-link}
+        from vizro import Vizro
+        import vizro.plotly.express as px
+        import vizro.models as vm
+
+        iris = px.data.iris()
+
+        page_1 = vm.Page(
+            title="My first page",
+            components=[
+                vm.Graph(figure=px.scatter(iris, x="sepal_length", y="petal_width", color="species")),
+            ],
+        )
+        page_2 = vm.Page(
+            title="My second page",
+            components=[
+                vm.Card(text="My text here"),
+            ],
+        )
+        page_3 = vm.Page(
+            title="My third page",
+            components=[
+                vm.Graph(figure=px.scatter(iris, x="sepal_length", y="sepal_width", color="species")),
```

Comment:
Maybe simplify the example and just have cards as components? You can just add emojis and different text to the card then to differentiate the pages :)

Reply:
If you want to do this, it's fine, but let's make a separate PR as I'm really just making the pycafe additions in this one.